### DISCUSSION TITLE
Meeting agenda 31st August

### DISCUSSION THREAD

**Discussion by ColinEberhardt** [finos/technical-oversight-committee#28] [Meeting Agenda]
Hi All,

Here is a proposed agenda for our next meeting:

- Can we agree / finalise our [contribution principles](https://github.com/finos/technical-steering-committee/discussions/19)?
  - if so, I'd be happy to formalise them in a brief write-up
- I'd like to make this repo public, is everyone happy to proceed with that by the end of this week?
- What do we want to tackle next as part of refining the contribution process? We discussed some other potential areas of improvement in an [earlier meeting](https://github.com/finos/technical-steering-committee/blob/master/meetings/2022-06-16.md).
- The FINOS team have [started work on a backlog](https://github.com/orgs/finos/projects/39), I'll work with them to elaborate on this further.

**Comment by jonfreedman** [comment]
On the topic of contribution principles - I am aware that you wrote most of them and I'm the only other person to add one - I think they are good but I am not sure if everyone has reviewed them all - this feels like a decentralized vote.

My only question about going public is if that means public read-only or read-write? I think read-write may be sensible given we still have our private mailing list.

Next steps - do we want some sort of holistic review of the existing project ecosystem? Is that our remit or not? The Rosetta contribution was interesting because it talked about working with Legend and Morphir - figuring out if the TSC can review/facilitate that level of interaction feels high value. I would also be curious if any of the existing projects have things they would like from us? (architecture/code/roadmap reviews for example).

**Reply by ColinEberhardt** [reply]
> On the topic of contribution principles - I am aware that you wrote most of them and I'm the only other person to add one - I think they are good but I am not sure if everyone has reviewed them all - this feels like a decentralized vote.

That's a fair point, and I am conscious of that too. Do you have any suggestions for how we can ensure that they reflect the collective wisdom of the group?

> My only question about going public is if that means public read-only or read-write? I think read-write may be sensible given we still have our private mailing list.

That's a good question. The git repo would be read-only, with write via pull-request. For discussions, I've read the GitHub documentation and I don't think we have much option other than to make it read/write (which is no great surprise, this is the same for issues). I'm happy to go with that, but we should discuss further.

Also, thanks for your suggestions on next steps. I've asked @mindthegab  and @jgavronsky whether the FINOS team or GB can provide a backlog of work too.

**Reply by jonfreedman** [reply]
I'd suggest an offline poll of all the members asking them if they have:

1. Read all the principles, have nothing to add
2. Read all the principles, have some feedback
3. Not read all the principles

I don't think it matters if bucket 3 is non-empty - but I feel like we probably want it to be <50% in order to ratify the first version of the principles, ideally we don't touch them again this year once they are ratified.

**Comment by stephengoldbaum** [comment]
Can we achieve with +1 vote in critical mass (and a reminder if we don't
teach it)? There's been ample opportunity and discussion.

P.S.,
I will miss tomorrow's meeting.

**Reply by ColinEberhardt** [reply]
Thanks Stephen, good news, critical mass was reached and the contribution principles finalised:

https://github.com/finos/technical-steering-committee/pull/36

we’re now applying them to Vuu and Desktop Agent Bridging
